modules/ROOT/pages/manage-proxy.adoc

```diff
@@ -31,29 +31,18 @@ The output should match the value expected.
 
 == Update the HTTP Proxy
 
-This operation requires two procedures: 
+Updating the HTTP proxy causes each machine in the cluster to restart all pods running on it. Although no downtime occurs if you are running multiple replicas, you should perform HTTP proxy updates during a maintenance window.
```

Comment:
How does the second sentence relate to the first (restarting all pods, downtime with multiple replicas)?
Replicas of what?